tests/e2e/create_module/kyma_create_module_test.go

```diff
@@ -66,25 +64,30 @@ func Test_ModuleTemplate(t *testing.T) {
 	sourceAccessSpec, err := ocm.DefaultContext().AccessSpecForSpec(source.Access)
 	assert.Nil(t, err)
 	githubAccessSpec, ok := sourceAccessSpec.(*github.AccessSpec)
+	assert.Equal(t, ok, true)
 	assert.Equal(t, githubAccessSpec.Type, github.Type)
 	assert.Contains(t, testRepoURL, githubAccessSpec.RepoURL)
 
 	// test security scan labels
 	secScanLabels := descriptor.Sources[0].Labels
 
 	var devBranch string
-	yaml.Unmarshal(secScanLabels[1].Value, &devBranch)
```

Comment:
Should we replace these identical labels validations with more descriptive
```go
assert.Equal(t, map[string]string{
    "branch":  "main",
    "tag":     "0.5.0",
    "lang":    "golang-mod",
    "exclude": "**/test/**,**/*_test.go", 
}, flatten(secScanLabels))
```

And a utility function `flatten` that abstracts-out yaml.Unmarshal operation on the value fields for the labels collection?